index.html

```diff
@@ -165,5 +165,21 @@ <h4 class="end-text">LETS'S CONNECT</h4>
             <hr>
         </article>
     </main>
+    <footer class='container'>
+        <article >
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xayvwlyd" method="post" class="contacts-form">
+                <label for="username"></label>
```

Comment:
- [ ] I'd suggest you remove all the label tags in the form since they are functional in this case since the placeholder attribute in the input tag already instructs the user on the input required. Also, this will help keep your code clean and more readable.